backend/app/api/endpoints/users.py

```
@@ -80,6 +80,26 @@ async def update_current_user_endpoint(
         raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail=str(e))
 
 
+@router.put("/me/password", response_model=UserInDB)
+async def change_password(
+    request: ChangePasswordRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.15.4)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)